metsrw/mets.py

```diff
@@ -505,6 +504,26 @@ def _parse_tree(self, tree=None):
                 file_uuid=entry.derived_from, type="Item"
             )
 
+    def _parse_header(self, tree):
+        header = self.tree.find(u"mets:metsHdr", namespaces=utils.NAMESPACES)
+        # Check CREATEDATE < now
+        if header is not None:
+            createdate = header.get(u"CREATEDATE")
+        else:
+            createdate = None
+        now = datetime.utcnow().isoformat(u"T")
```

Comment:
Py2: `TypeError: isoformat() argument 1 must be char, not unicode`

:smile: :smile: :smile: :smile:

Reply:
_le sigh_

I like how in py3, it gives a TypeError if you pass bytes. Oh well, left as "T", that works in both.